Formula/maven@3.0.rb

```diff
@@ -0,0 +1,52 @@
+class MavenAT30 < Formula
+  desc "Java-based project management"
+  homepage "https://maven.apache.org/"
+  url "https://archive.apache.org/dist/maven/maven-3/3.0.5/binaries/apache-maven-3.0.5-bin.tar.gz"
```

Comment:
This should be `https://www.apache.org/dyn/closer.cgi?path=maven/maven-3/3.0.5/binaries/apache-maven-3.0.5-bin.tar.gz`